### PR TITLE
MEN-5458 Monitor: tests: we expect only one CRITICAL for log subsystem.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
goes with 
https://github.com/mendersoftware/monitor-client/pull/117
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/493346162

**Comment by merlin-northern** [comment]
failure in the tests is unrelated.

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -1121,8 +1121,10 @@ def test_monitorclient_logs_and_services(self, monitor_commercial_setup_no_clien
         _, messages = get_and_parse_email_n(
             monitor_commercial_setup_no_client, user_name, 4
         )
-        assert messages_count >= 4
-        for m in [messages[-1], messages[-2]]:
+        messages_count = len(messages)
+        assert messages_count == 5
```
Comment (lluiscampos) [inline tests/tests/test_monitor_client.py:1125]: I think we are abusing this `message_count` variable. It is being re-defined in many places and prone to errors. I think we should remove it completely and do only:
```suggestion
        assert len(messages) == 5
```

We can also clean-up this messy block above where `message_count` is being redefined without even checking, and we have two asserts for the same thing:
image

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Please squash both commits, both are changing the two tests and it is not clear that the first one makes sense alone.

File: tests/tests/test_monitor_client.py (hunk 1 above)
Comment (merlin-northern) [reply tests/tests/test_monitor_client.py]: cleaned-fixed-squashed. thanks.

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**